packages/manganis/manganis-core/src/css.rs

```diff
@@ -21,35 +21,60 @@ pub struct CssAssetOptions {
 
 impl Default for CssAssetOptions {
     fn default() -> Self {
-        Self::new()
+        Self::default()
     }
 }
 
 impl CssAssetOptions {
     /// Create a new css asset using the builder
-    pub const fn new() -> Self {
+    #[deprecated(note = "Use `AssetOptions::css()` instead", since = "0.7.0")]
+    pub const fn new() -> AssetOptionsBuilder<CssAssetOptions> {
```

Comment:
On the fence about deprecating this. It no longer returns Self which is odd, but this is probably what you want if you try to build `CssAssetOptions`. Keeping the new method makes this change much less breaking